### PR TITLE
Makes mechtopic look nice.

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

As per title. Not a lot of actual design changes mostly just makes the code look *better*.
The spoilers have a little more distance between each other, otherwise looks and functions almost the same.
Also, replaces a spawn with a callback.
Also makes it possible to set the internal pressure to exactly zero.
Also adds some close buttons in Topic.
Also fixes the logic behind DNA-locks. (nulls aren't known for having DNA).
Also adds a bunch of early returns in Topic.

## Why It's Good For The Game

This isn't good but at least it's readible.

## Changelog
:cl:
tweak: Some small changes to the mech HMTL windows.
code: Changed a lot of code around mech topic.
/:cl:

### CODE REVIEW

**Comment by nemvar** [comment]
ui
In case you were wondering, new thing almost looks exactly like old thing.

**Comment by Tlaltecuhtli** [comment]
how did you make the window look like its from win 7?

**Comment by nemvar** [comment]
by using windows 7

**Comment by Dennok** [comment]
> how did you make the window look like its from win 7?

Set in config, window mode with frills.

File: code/game/mecha/mecha_topic.dm [hunk 1]
```diff
@@ -316,26 +364,34 @@
...
-			if(T == loc)
-				clearInternalDamage(MECHA_INT_CONTROL_LOST)
-				occupant_message("<span class='notice'>Recalibration successful.</span>")
-				log_message("Recalibration of coordination system finished with 0 errors.", LOG_MECHA)
-			else
-				occupant_message("<span class='warning'>Recalibration failed!</span>")
-				log_message("Recalibration of coordination system failed with 1 error.", LOG_MECHA, color="red")
+		addtimer(CALLBACK(src, .proc/stationary_repair, loc), 100, TIMER_UNIQUE)
+
+//Repairs internal damage if the mech hasn't moved.
```
Comment (ShizCalev) [inline code/game/mecha/mecha_topic.dm]: one more slash for autodocs

**Pull request merged**